src/Commands/PollingScanStatus.ts

```diff
@@ -77,11 +77,12 @@ export class PollingScanStatus implements CommandModule {
         logger.error(`The breakpoint has been hit during polling.`);
         logger.error(`Breakpoint: ${e.message}`);
         process.exit(50);
-
-        return;
       }
 
-      logger.error(`Error during "scan:polling": ${e.error || e.message}`);
+      const message = Helpers.hasErrorProperty(e)
+        ? e.error
+        : Helpers.getErrorMessage(e);
```

Comment:
fine grained `Helpers.hasErrorProperty` seems redundant. `Helpers.getErrorMessage` can handle multiple sources - `message`, `error`, etc.

Reply:
Type assertion can return just one type. I can't assert one function different types.

Reply:
think about purpose of your `getErrorMessage` helper. imo it should take error in any form and return error message `string` as result - no matter how. it can be easily extended to cover "message in `error` field" case to make the usage code cleaner, and avoid having any extra verbose `hasX` helpers.

Another point i forgot to mention initially - is current logic of `getErrorMessage`, its type checking part. imo strict type checking is not required here, it should be compatible with non-Error instances like plain `{message: 'Custom Error'}` objects.